doc/Learn/Maui/ThirdParty-GrapeCity.md

```diff
@@ -1,17 +1,17 @@
 ---
-uid: Uno.Extensions.Maui.ThirdParty.GrapeCity
+uid: Uno.Extensions.Maui.ThirdParty.ComponentOne
```

Comment:
@vanslyker thank you for the PR!
 If the uid is changed here, it will need to be updated here as well please
 https://github.com/unoplatform/uno.extensions/blob/3ae15349b1a0c33ce0b6c5a04baeddf5632d0254/doc/Learn/Maui/toc.yml#L9